prow/cmd/tide/config.md

```diff
@@ -44,6 +44,7 @@ Each query corresponds to a set of **open** PRs as candidates for merging.
 It can consist of the following dictionary of fields:
 
 * `orgs`: List of queried organizations.
+  * `excludedRepos`: List of ignored repositories.
```

Comment:
Seems there's extra blanks here? Can you align with other lines

Reply:
Label `excludedRepos` is the sublist of `orgs`, so these extra 2 black is to show words like this:
* `orgs`: List of queried organizations.
  * `excludedRepos`: List of ignored repositories.

Reply:
My mistake. It seems that the `orgs` and `excludedRepos` are at the same level. I've correct the mistake...